.vib/appsmith/vib-publish.json

```diff
@@ -0,0 +1,85 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/appsmith"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/appsmith"
+        },
+        "runtime_parameters": "YmFja2VuZDoKICBhZG1pblBhc3N3b3JkOiAiYml0bmFtaSExMjM0IgogIGNvbnRhaW5lclBvcnRzOgogICAgaHR0cDogODgyMwogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgaHR0cDogODA4MwoKCnJ0czoKICBjb250YWluZXJQb3J0czoKICAgIGh0dHA6IDg4NTAKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgwODUKY2xpZW50OgogIHNlcnZpY2U6CiAgICB0eXBlOiBMb2FkQmFsYW5jZXIKICAgIHBvcnRzOgogICAgICBodHRwOiA4MAogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIHJ1bkFzVXNlcjogMTAwMgogIHBvZFNlY3VyaXR5Q29udGV4dDoKICAgIGZzR3JvdXA6IDEwMDIKCg==",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "L4"
```

Comment:
After performing several tests using machine M4, the deployment phase fails sometimes because of the containers not being ready. I moved back to L4 which consistently works.